### DISCUSSION TITLE
Meeting Feed limit per page

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1317] [Q&A] [answered]
Hi there,

I was wondering if there is an option to have a limit to how many meetings load per page (i.e. 30 meetings) and then have a Load More button to reveal more meetings or pages like on blog pages? 

The reason I'm asking is that on the Find a Meeting page of our website I've linked on the menu dropdown "[Meetings by State](https://al-anon.org.au/meetings/#meeting-by-state)" so members can navigate to their state on the page and download a copy of meetings in their state. 

The problem occurs when they click on this link, the webpage goes to the location on the page where this information is sitting but it gets pushed down when the meeting feed loads.

For the moment I've added these links to the menu under "Meetings from State" to remove the frustration.

View these links below for an example, and an image attached of the meeting by state area:
(https://al-anon.org.au/meetings/)
(https://al-anon.org.au/meetings/#meeting-by-state)

![Screen Shot 2024-01-22 at 11 03 39 am](https://github.com/code4recovery/12-step-meeting-list/assets/23748349/51dd0d57-8733-4063-b326-d63236a0f1f4)

**Comment by joshreisner** [comment, marked as answer]
hi @nataliefaith2 - TSML UI is not designed for content to go below the list. i'd recommend creating a separate page for the PDF downloads. you could still link to it below the "Find a Meeting" dropdown.

TMSL UI does display meetings 10 at a time, but displays the next 10 automatically when you reach the bottom of the page. this is easier for users than making them tap a button to show the next set
